components/AllocatedWorkers/AllocatedWorkers.spec.tsx

```diff
@@ -32,23 +33,30 @@ describe(`AddAllocatedWorker`, () => {
     }));
 
   const props = {
-    id: 123,
+    person: mockedResident,
   };
 
   it('should render only the table if not allocator', async () => {
```

Comment:
Could we rephrase this test? e.g. should only render allocated workers table if the user does not have allocation permissions?

Reply:
Great shout – I didn't change this test as part of this PR but it is an awful name. I will update it while I'm here 😄

Reply:
Done in https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/354/commits/bede9b7b84626171026a63095779d48f6f00561c

Reply:
🙌🏾